src/OrderStates.php

```diff
@@ -237,4 +237,22 @@ private function setStateIcons($state, $orderStateId)
 
         return true;
     }
+
+    /**
+     * Check if OrderState already exists in the table
+     *
+     * @param int $orderStateId
+     *
+     * @return bool
+     */
+    private function stateAlreadyExists($orderStateId)
+    {
+        $query = new \DbQuery();
+        $query->select('id_order_state');
+        $query->from(static::ORDER_STATE_TABLE);
```

Comment:
Maybe self:: should be used instead? Since it's used everywhere currently

Reply:
done